src/video/x11/SDL_x11vulkan.c

```diff
@@ -39,6 +39,13 @@
 #define DEFAULT_X11_XCB "libX11-xcb.so.1"
 #endif
 
+SDL_ELF_NOTE_DLOPEN(
+    "x11-vulkan",
+    "Support for vulkan on X11",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_SUGGESTED,
+    DEFAULT_VULKAN, DEFAULT_X11_XCB
+);
```

Comment:
This should maybe be two records both labelled `"x11-vulkan"`, one for `libvulkan` and one for `libX11-xcb`?